draft-ietf-tls-tls13.md

```diff
@@ -2376,7 +2376,15 @@ RSASSA-PSS algorithms
   digest used in the mask generation function and the digest being signed are
   both the corresponding hash algorithm as defined in {{!SHS}}. When used in
   signed TLS handshake messages, the length of the salt MUST be equal to the
-  length of the digest output.  This codepoint is new in this document and is also
+  length of the digest output. When used in certificate signatures, the
+  length of the salt MUST be at least as long as the length of the
+  digest output, and the algorithm parameters MUST be DER encoded.
+  This codepoint indicates that the sender
```

Comment:
There is more than one RSASSA-PSS codepoint, so "these" is probably more appropriate.

Reply:
which codepoint you refer to? there is only rsassaPss OID. What's variable is the parameters field in the Algorithm Identifier struct

Reply:
sha256, sha384, etc.